### ISSUE TITLE
Weird Hint behavior on Android

### ISSUE THREAD

**Issue by jakemadash** [wix/react-native-ui-lib#3913]
## Description

On Android, the component wrapped by Hint shifts position when pressed. And then, when visibility is set back to false, an error like the below is thrown:

`sendAccessibilityEvent() dropping event: Cannot find view with tag #8446`

### Related to

- [x] Components
- [ ] Demo
- [ ] Docs
- [ ] Typings

### Steps to reproduce

1. Use the Hint component on Android and observe the behavior when toggling visibility of the hint's contents.

### Code snippet

```
        <Hint
          visible={hintVisible}
          color="white"
          enableShadow
          offset={6}
          borderRadius={8}
          containerWidth={400}
          removePaddings
          onBackgroundPress={() => setHintVisible(!hintVisible)}
          customContent={
            <Text className="text-xs text-center p-2.5 px-5">
              Your ability to submit bids, update membership, and accept offers has been disabled.
              Where applicable, all your max bids have been lowered to current bids. Please contact
              Customer Support:{' '}
              <STText classes="color-[#6CA5C8]" onPress={openSubmitRequest}>
                submit a request
              </STText>
              .
            </Text>
          }
        >
          <Pressable
            accessibilityRole="button"
            accessibilityLabel="info"
            accessibilityHint="opens dialog explaining account suspension"
            onPress={() => setHintVisible(!hintVisible)}
          >
            <FontAwesomeIcon icon={faCircleInfo} size={18} color={'white'} />
          </Pressable>
        </Hint>
```

### Screenshots/Video

https://github.com/user-attachments/assets/1996adf7-2a87-4256-8e3f-d6a74e062383

### Environment

- React Native version: 0.81.5
- React Native UI Lib version: 8.2.1

### Affected platforms

- [x] Android
- [ ] iOS
- [ ] Web

**Comment by melanymunozv** [comment]
me pasa lo mismo
